### DISCUSSION TITLE
Playground  view  DataPreview  error，its about the  prefix of default_cluster  ，how to solve it

### DISCUSSION THREAD

**Discussion by xiaoyunjie** [apache/doris#33207] [A - General / Q&A]
![699288b750cae918a98a70f83e59acee](https://github.com/apache/doris/assets/18340924/c8c39973-6f34-46a8-af7b-67096e26a77e)
![cd4c6cd2db65b17da456a9aea9c9c450](https://github.com/apache/doris/assets/18340924/ff53ea17-bcfb-4087-8d8c-39c7dd0bf04f)
![967940e3d475e9592d4252da3bbe67c7](https://github.com/apache/doris/assets/18340924/4352f251-75ac-4867-b046-4b8ef1f6aa8b)

**Comment by yagagagaga** [comment]
Don't use playground. Recommend using dbeaver.
